plugins/inputs/fibaro/fibaro.go

```diff
@@ -0,0 +1,183 @@
+package fibaro
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/httpjson"
+)
+
+const sampleConfig = `
+  ## Required Fibaro controller address/hostname.
+  ## Note: connection is done over http/80 as Fibaro did not implement https.
+  server = "<controller>"
```

Comment:
I think we should use an origin style URL: `http://hostname:port`, this way if Fibaro does the right thing and adds https we can easily add support.